driver-core/src/main/com/mongodb/internal/connection/BaseCluster.java

```diff
@@ -265,6 +265,10 @@ public ClusterDescription getCurrentDescription() {
         return description;
     }
 
+    private synchronized void updatePhase() {
+        phase.getAndSet(new CountDownLatch(1)).countDown();
```

Comment:
Can this method also be used in `updateDescription`?

Reply:
Yes, will fix and merge.